config/initializers/datadog.rb

```diff
@@ -9,6 +9,7 @@
     # Namespace our app
     c.service = 'vets-api'
     c.env = Settings.vsp_environment unless ENV['DD_ENV']
+    c.version = AppInfo::GIT_REVISION unless ENV['DD_VERSION']
```

Comment:
I'm guessing we don't also need this on the `master` branch of this file as well?

Reply:
We will definitely need this in master as well.